l10n_it_riba/models/riba.py

```diff
@@ -123,6 +129,10 @@ def _compute_payment_ids(self):
         default=lambda self: fields.Date.context_today(self),
         help="Keep empty to use the current date.",
     )
+    total_amount = fields.Float(
+        string="Amount",
+        compute="_compute_total_values",
```

Comment:
Sarebbe meglio chiamare il metodo `_compute_total_amount` così come indicato nelle [convezioni OCA](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#models). Nota: considerato che vuoi mantenere tutti i commit tale modifica dovrebbe rientrare in https://github.com/OCA/l10n-italy/pull/3984/commits/9e6110af80f3243c2dfd0fcb244c808768cf2ac7
Grazie.